test/audit/agent-test-tracing-journey.ts

```
@@ -0,0 +1,276 @@
+    }),
+  });
+
+  sdk.registerTool("calculate", {
+    description: "Calculate a math expression",
+    parameters: {
+      type: "object" as const,
+      properties: { expression: { type: "string" } },
+      required: ["expression"],
+    },
+    execute: async (args: Record<string, unknown>) => ({
+      result: eval(String(args.expression)),
```

Biome (2.4.4)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)